app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/CalendarItem.kt

```diff
@@ -21,19 +21,7 @@ enum class CalendarItemType {
  * Entity for storing information about lecture events
  */
 @Entity(tableName = "calendar")
-data class CalendarItem(
-    @PrimaryKey
-    var nr: String = "",
-    var status: String = "",
-    var url: String = "",
-    var title: String = "",
-    var description: String = "",
-    var dtstart: DateTime = DateTime(),
-    var dtend: DateTime = DateTime(),
-    var location: String = "",
-    @Ignore
-    var blacklisted: Boolean = false
-) : WeekViewDisplayable<CalendarItem> {
+data class CalendarItem(@PrimaryKey var nr: String = "", var status: String = "", var url: String = "", var title: String = "", var description: String = "", var dtstart: DateTime = DateTime(), var dtend: DateTime = DateTime(), var location: String = "", @Ignore var blacklisted: Boolean = false) : WeekViewDisplayable<CalendarItem> {
```

Comment:
That is a lot of content for one line.
Is this mandated by another tool?

Reply:
I've split it back to multiple lines, my android studio configuration used too many spaces in the standard configuration which led to the faulty indentation